### PR TITLE
[Fix] Updates copy for GC Digital Talent in French

### LINKED ISSUE #6710: 🐛 GC Digital Talent inconsistently translated in French
## 🐛 Bug

**GC Digital Talent** is translated in French differently in our codebase:

1. Talent numérique du GC
2. Talents numériques du GC

## 🦋 Expected Behaviour

**GC Digital Talent** should be translated in French the same way.

## 📸 Screenshot
Screen Shot 2023-05-17 at 10 22 53

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] **GC Digital Talent** is translated in French as "Talents numériques du GC"
- [ ] Also in TC report

### PR DESCRIPTION
**Pull request by mnigh**
🤖 Resolves #6710.

## 👋 Introduction

This PR updates instances of _GC Digital Talent_ in French to all be the same: _Talents numériques du GC_.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. `npm run build`
3. Verify no instances of _Talent numérique du GC_
4. Build tc-report
5. Verify no instances of _Talent numérique du GC_

### CODE REVIEW

**Comment by esizer** [comment]
Does the scope of this include where we reference just "Digital Talent" and not the entire "GC Digital Talent"?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/852b269cce352d46306b26e144d26ff737ff4b6e/apps/web/src/lang/fr.json#L812-L815

**Comment by mnigh** [comment]
> Does the scope of this include where we reference just "Digital Talent" and not the entire "GC Digital Talent"?
> 
> https://github.com/GCTC-NTGC/gc-digital-talent/blob/852b269cce352d46306b26e144d26ff737ff4b6e/apps/web/src/lang/fr.json#L812-L815

Great question, I'm not sure! @gobyrne, any thoughts?

**Comment by gobyrne** [comment]
@Jerryescandon We'll need to update designs with this copy as well. Can you tackle it? If not, please pass it to Substrae

File: apps/web/src/lang/fr.json [hunk 1]
```diff
@@ -2293,16 +2285,16 @@
...
     "description": "Opening paragraph for accessibility statement"
   },
   "vC9NsS": {
     "defaultMessage": "Nos normes",
     "description": "Title for the accessibility standards"
   },
-  "a3c78j": {
-    "defaultMessage": "L’équipe de Talent numérique <abbreviation>GC</abbreviation> adhère aux <digitalStandardsLink>normes numériques inclusives du gouvernement du Canada</digitalStandardsLink> (GC).",
+  "tu2Z17": {
+    "defaultMessage": "L’équipe de Talent numérique GC adhère aux <digitalStandardsLink>normes numériques inclusives du gouvernement du Canada</digitalStandardsLink> (GC).",
```
Comment (esizer) [inline apps/web/src/lang/fr.json:2297]: Should this also be updated to:

```suggestion
    "defaultMessage": "L’équipe de Talents numériques GC adhère aux <digitalStandardsLink>normes numériques inclusives du gouvernement du Canada</digitalStandardsLink> (GC).",
```

**Review by esizer — CHANGES_REQUESTED** [review]
Feel free to dismiss because I have no idea what I'm talking about here but this is the last instance of _Talent numérique_ I could find.

File: apps/web/src/lang/fr.json (hunk 1 above)
Comment (mnigh) [reply apps/web/src/lang/fr.json]: Almost, good catch though. How I missed including my own suggestion is perplexing, but somehow, I did. 🤷 

Updated in 77e3e8dcaeccf36abd584840e011b40ed17767c1

**Review by esizer — APPROVED** [review]

**Review by esizer — APPROVED** [review]
Looking good 💅

**Pull request merged**